bci_tester/data.py

```diff
@@ -199,8 +199,11 @@ def create_container_version_mark(
     BCI_DEVEL_REPO = f"https://updates.suse.com/SUSE/Products/SLE-BCI/{OS_MAJOR_VERSION}-SP{OS_SP_VERSION}/{LOCALHOST.system_info.arch}/product/"
     _BCI_REPLACE_REPO_CONTAINERFILE = ""
 else:
-    _BCI_REPLACE_REPO_CONTAINERFILE = f"RUN sed -i 's|baseurl.*|baseurl={BCI_DEVEL_REPO}|' /etc/zypp/repos.d/{BCI_REPO_NAME}.repo"
+    bci_repo_path = f"/etc/zypp/repos.d/{BCI_REPO_NAME}.repo"
+    # only try to sed the baseurl if we can write to the repo
+    _BCI_REPLACE_REPO_CONTAINERFILE = f"RUN if [ -w {bci_repo_path} ]; then sed -i 's|baseurl.*|baseurl={BCI_DEVEL_REPO}|' {bci_repo_path}; fi"
 
+assert BCI_DEVEL_REPO, "BCI_DEVEL_REPO must be set at this point"
```

Comment:
I don't understand this assert. isn't that always wrong for LTSS containers?

Reply:
The variable is set to a value in line 199 if it is set to `None`.

Reply:
yeah, thanks. a 2nd look helped. sorry for the false noise.